src/pages/Dashboard/AdminDashboard.module.css

```
@@ -241,7 +267,7 @@
     text-decoration: underline;
 }
 
-/* ── Match table ─────────────────────────────────────── */
+/* Match Table*/
```

Stylelint (17.9.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)